src/cli/cli.ts

```diff
@@ -892,7 +918,7 @@ async function closeWithError(error: any, args: CommandArgs, showHelpIfEnabled:
   let errorMessage: string = error instanceof CommandError ? error.message : error;
 
   if (error instanceof ZodError) {
-    errorMessage = error.errors.map(e => `${e.path}: ${e.message}`).join(os.EOL);
+    errorMessage = error.errors.map(e => `${e.path.join('.')}: ${e.message}`).join(os.EOL);
```

Comment:
@waldekmastykarz why do we need to do ``${e.path.join('.')}:` here? 

From what I experimented with `login` and `spo site get` command this is always empty and due to that we get a strange error message with `Error: : (...)` message. This happens only to commands that were refactored with using zod

![image](https://github.com/user-attachments/assets/168fa4b1-0ec8-42a1-a8c8-394c6ee10c10)

Reply:
zod allows for passing the path of the faulty object in the exception which theoretically can be an array, hence the construct. I think we should have another look at the zod errors we throw so that we properly use this feature.

Reply:
I agree. What if we just check if the path is not empty and then present it along with the additional `:`, and if it is empty we don't print it and also don't print the `:` so we don't end up with `:  :` kinda strange thing?
Would that work for you? If you are busy I may do a fixup up to that and merge it today.

Reply:
I updated the PR. Basically the "rule" is: when you validate a single property, zod automatically appends the property name to the error message. When you validate multiple properties, you're responsible for setting the path property on the error message to make it clear to the user which property should be fixed.

Reply:
ok, I will recheck it today